stylus/include/Home.styl

```diff
@@ -17,16 +17,17 @@
 		padding-bottom: 3rem
 	>div:nth-child(5), div:nth-child(6)
 		display: flex
-		justify-content: center
+		justify-content: space-evenly
 		padding-bottom: 1rem
+		padding-left: 5rem
+		padding-right: 5rem
```

Comment:
I actually think the rows would look better without any bottom padding. I'd also get rid of left and right here, but add a small bit of padding (1-2rems) on the left and right of the view's container `.Home`. Then you just let the flexbox work its magic.